### PR TITLE
Change Find domain links in all of our services

### PR DESCRIPTION
**Pull request by tomas-stefano**
### Context

The Find domain has change.

We should point to the new domain.

### Guidance to review

1. All links pointing to find are updated to use the new domain?
2. Is the content security policy working?
3. Is the maintenance page working?

### CODE REVIEW

**Comment by tomas-stefano** [comment]
[Change Find domain links in all of our services](https://trello.com/c/FGS5l41G/2065-change-find-domain-links-in-all-of-our-services)

**Comment by tomas-stefano** [comment]
This PR doesn't touch on the utm mentions from GiT links. We aligned with GiT team and this will be done in a separate work as it needs improvement the GiT linking.

**Review by dcyoung-dev — APPROVED** [review]
A question for another day but do we still need all the files in the `settings/` directory?

File: service_unavailable_page/web/nginx.conf [hunk 1]
```diff
@@ -41,7 +41,7 @@ http {
 
   server {
     listen       {{port}};
-    server_name  www.find-postgraduate-teacher-training.service.gov.uk qa.find-postgraduate-teacher-training.service.gov.uk;
+    server_name  www.find-teacher-training-courses.service.gov.uk qa.find-teacher-training-courses.service.gov.uk;
```
Comment (tomas-stefano) [inline service_unavailable_page/web/nginx.conf:44]: This should be without www correct?

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 This repo is home to three services:
 
-- A service for candidates to [find teacher training](https://www.find-postgraduate-teacher-training.service.gov.uk)
+- A service for candidates to [find teacher training](https://www.find-teacher-training-courses.service.gov.uk)
```
Comment (saliceti) [inline README.md:5]: No www

File: README.md [hunk 3]
```diff
@@ -23,9 +23,9 @@ This repo is home to three services:
 
 | Name        | URL                                                                    | Description
 | ----------- | ---------------------------------------------------------------------- | ------------------------------------------------------------------------------
-| Production  | [www](https://www.find-postgraduate-teacher-training.service.gov.uk)   | Public site
-| Staging     | [staging](https://staging.find-postgraduate-teacher-training.service.gov.uk)| For internal use by DfE to test deploys
-| QA          | [qa](https://qa.find-postgraduate-teacher-training.service.gov.uk)     | For internal use by DfE for testing. Automatically deployed from main
+| Production  | [www](https://www.find-teacher-training-courses.service.gov.uk)   | Public site
```
Comment (saliceti) [inline README.md:26]: No www

File: config/initializers/content_security_policy.rb [hunk 4]
```diff
@@ -10,6 +10,7 @@
   config.content_security_policy do |policy|
     local_domains = %w[https://publish.localhost https://find.localhost]
     prod_domains = [
+      'https://*.find-teacher-training-courses.service.gov.uk',
```
Comment (saliceti) [inline config/initializers/content_security_policy.rb:13]: Does this match 'https://find-teacher-training-courses.service.gov.uk/' ?

File: config/settings/test.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@ govuk_notify:
 system_authentication_token: "Ge32"
 gcp_api_key: please_change_me
 publish_url: https://www.publish-teacher-training-courses.service.gov.uk
-find_url: https://www2.find-postgraduate-teacher-training.service.gov.uk
+find_url: https://www2.find-teacher-training-courses.service.gov.uk
```
Comment (saliceti) [inline config/settings/test.yml:6]: No www2

File: service_unavailable_page/web/nginx.conf (hunk 1 above)
Comment (saliceti) [reply service_unavailable_page/web/nginx.conf:44]: Yes: no www

File: config/settings/test.yml (hunk 5 above)
Comment (tomas-stefano) [reply config/settings/test.yml]: Removed

File: public/robots.txt [hunk 6]
```diff
@@ -1,5 +1,5 @@
 # See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
 User-agent: *
 Disallow: /
-Allow: https://www.find-postgraduate-teacher-training.service.gov.uk/sitemap.xml
-Sitemap: https://www.find-postgraduate-teacher-training.service.gov.uk/sitemap.xml
+Allow: https://find-teacher-training-courses.service.gov.uk/sitemap.xml
+Sitemap: https://www.find-teacher-training-courses.service.gov.uk/sitemap.xml
```
[the remark this answers is not in the document]
Comment (tomas-stefano) [reply public/robots.txt]: Removed

File: config/initializers/content_security_policy.rb (hunk 4 above)
Comment (tomas-stefano) [reply config/initializers/content_security_policy.rb:13]: Yes it does from my testing on the review and local app

File: README.md (hunk 3 above)
Comment (tomas-stefano) [reply README.md]: Removed

File: README.md (hunk 2 above)
Comment (tomas-stefano) [reply README.md]: Removed

File: service_unavailable_page/web/nginx.conf (hunk 1 above)
Comment (tomas-stefano) [reply service_unavailable_page/web/nginx.conf]: Done

**Review by saliceti — APPROVED** [review]

**Review by avinhurry — COMMENTED** [review]
Do we need to change these: 

- https://github.com/DFE-Digital/publish-teacher-training/blob/4ee08000987054785fa6397e2acba18fd5f415d8/config/locales/find.yml#L244

**Comment by tomas-stefano** [comment]
> Do we need to change these:
> 
> * https://github.com/DFE-Digital/publish-teacher-training/blob/4ee08000987054785fa6397e2acba18fd5f415d8/config/locales/find.yml#L244

No, I mentioned here about for more details https://github.com/DFE-Digital/publish-teacher-training/pull/4397#issuecomment-2245325537

**Review by avinhurry — APPROVED** [review]

**Pull request merged**